mobile/android/app/src/main/res/layout/slideshow_layout.xml

```diff
@@ -1,27 +1,81 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
     android:id="@+id/widget_container">
+
+    <!-- Main Image (if available) -->
     <ImageView
-        android:id="@+id/widget_placeholder"
+        android:id="@+id/widget_img"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scaleType="fitCenter"
-        android:adjustViewBounds="true"
-        android:visibility="visible"
-        tools:visibility="visible"
-        />
-    <ImageView
-        android:id="@+id/widget_img"
+        android:scaleType="centerCrop"
+        android:visibility="gone" />  <!-- Initially hidden -->
+
+    <!-- Gradient Overlay for Text Readability -->
+    <LinearLayout
+        android:id="@+id/widget_overlay"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom"
+        android:orientation="vertical"
+        android:paddingStart="16dp"
+        android:paddingEnd="16dp"
+        android:paddingBottom="16dp"
+        android:paddingTop="4dp"
+        android:background="@layout/gradient_overlay"
+        android:visibility="gone"> <!-- Initially hidden, shown when image is available -->
+
+        <!-- Title -->
+        <TextView
+            android:id="@+id/widget_title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Where is the title?"
```

Comment:
Should we keep default text as empty here?

Reply:
Yes, that will be better. These were just test placeholders afterall.